geode-core/src/main/java/org/apache/geode/internal/cache/wan/GatewayReceiverMeters.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.internal.cache.wan;
+
+import static java.lang.String.format;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+
+import org.apache.geode.Statistics;
+
+public class GatewayReceiverMeters {
+  private static final Set<String> meterWhitelist = new HashSet<String>() {
+    {
+      add("cache.gatewayreceiver.events.received");
+    }
+  };
+
+  private final Counter eventsReceivedCounter;
+
+  public GatewayReceiverMeters(MeterRegistry meterRegistry, GatewayReceiverStats stats) {
+    eventsReceivedCounter = createIntCounter(meterRegistry, "cache.gatewayreceiver.events.received",
```

Comment:
Multiple copies of the same name, please you constants.

Reply:
Will do.